docker/llm/finetune/xpu/docker/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM intel/oneapi-basekit:2023.2.1-devel-ubuntu22.04
+ARG http_proxy
+ARG https_proxy
+ENV TZ=Asia/Shanghai
+ENV PIP_NO_CACHE_DIR=false
+
+RUN apt-get update && \
+    # install Intel GPU driver
+    apt-get install -y curl wget git gnupg gpg-agent && \
+    wget -qO - https://repositories.intel.com/graphics/intel-graphics.key | gpg --dearmor --output /usr/share/keyrings/intel-graphics.gpg && \
+    echo 'deb [arch=amd64,i386 signed-by=/usr/share/keyrings/intel-graphics.gpg] https://repositories.intel.com/graphics/ubuntu jammy arc' | tee /etc/apt/sources.list.d/intel.gpu.jammy.list && \
+    rm /etc/apt/sources.list.d/intel-graphics.list && \
+    # install python 3.9
+    ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone && \
+    env DEBIAN_FRONTEND=noninteractive apt-get update && \
+    apt install software-properties-common libunwind8-dev vim less -y && \
+    add-apt-repository ppa:deadsnakes/ppa -y && \
+    apt-get install -y python3.9 git curl wget && \
```

Comment:
wget, curl, git are duplicated with line 9
please use one command to update apt-get, 3 separated commands to intsall basic lib, gpu driver and py 39

Reply:
done